Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
Gotcha, kind of figured that was the example you were making.

I'm not sure on lint. I'm pretty sure that's one of the default targets in the gradle build, so probably. As I mentioned elsewhere, I'm not super familiar with the Android SDK -- I'm mostly the iOS guy. If we're not, we certainly should be.